docs/resources/Channel.md

```diff
@@ -629,22 +634,26 @@ Update a channel's settings. Requires the `MANAGE_CHANNELS` permission for the g
+| --------------------- | ----------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------- |
+| name                  | ?string                                                                 | 2-100 character channel name                                                                                                                                                    | Group DM, Text, News, Store, Voice, Category |
+| type                  | integer                                                                 | the [type of channel](#DOCS_RESOURCES_CHANNEL/channel-object-channel-types); only conversion between text and news is supported and only in guilds with the "NEWS" feature      | Text, News                                   |
+| position              | ?integer                                                                | the position of the channel in the left-hand listing                                                                                                                            | Text, News, Store, Voice, Category           |
+| topic                 | ?string                                                                 | 0-1024 character channel topic                                                                                                                                                  | Text, News                                   |
+| nsfw                  | ?boolean                                                                | whether the channel is nsfw                                                                                                                                                     | Text, News, Store                            |
+| rate_limit_per_user   | ?integer                                                                | amount of seconds a user has to wait before sending another message (0-21600); bots, as well as users with the permission `manage_messages` or `manage_channel`, are unaffected | Text                                         |
+| bitrate               | ?integer                                                                | the bitrate (in bits) of the voice channel; 8000 to 96000 (128000 for VIP servers)                                                                                              | Voice                                        |
+| user_limit            | ?integer                                                                | the user limit of the voice channel; 0 refers to no limit, 1 to 99 refers to a user limit                                                                                       | Voice                                        |
```

Comment:
what does not specifying this imply?

I can guess the answer, assuming a sane implementation; someone else might not be familiar with the default.